### PR TITLE
fix: use float instead of int in query results to avoid overflow

### PR DESCRIPTION
**Pull request by saravmajestic**
## Overview
- Fixes integer overflow in query results panel new UI
- perspective.js does not handle int64 and workaround suggested is to convert the int to float and use without precision. https://github.com/finos/perspective/issues/1346
- So for any integer type in schema, we will use float type with 0 precision digits and without grouping to avoid thousands separator

## Checklist

- [x] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: query_panel/main.js [hunk 1]
```diff
@@ -67,7 +69,7 @@ class Grid {
     await this.elem.load(table);
     await this.elem.restore({
       columns: [], // reset columns
-      settings: true,
+      settings: false,
```
Comment (mdesmet) [inline query_panel/main.js:72]: Why this change?

Comment (saravmajestic) [reply query_panel/main.js:72]: This is not intentional. good catch!

**Review by mdesmet — APPROVED** [review]
